admin/builders/centos64_build.sh

```
@@ -31,87 +31,26 @@ echo "The Builder $0 is building the Percolator packages with src=${src_dir} and
 whoami
 
 sudo dnf -y install gcc gcc-c++ wget rpm-build cmake
-sudo dnf install sqlite-devel zlib-devel bzip2-devel
-sudo dnf install tokyocabinet-devel xerces-c-devel
-sudo dnf install libtirpc libtirpc-devel
-sudo dnf -y --enablerepo=powertools install gtest
+sudo dnf -y install zlib-devel bzip2-devel
+sudo dnf -y --enablerepo=crb install gtest gtest-devel
+sudo dnf -y install boost-static boost-devel
 
 cd ${src_dir}
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)